pkg/kv/kvserver/batcheval/cmd_export.go

```diff
@@ -181,19 +181,27 @@ func evalExport(
 	var curSizeOfExportedSSTs int64
 	for start := args.Key; start != nil; {
 		destFile := &storage.MemFile{}
-		summary, resume, err := storage.MVCCExportToSST(ctx, cArgs.EvalCtx.ClusterSettings(), reader,
-			storage.MVCCExportOptions{
-				StartKey:           storage.MVCCKey{Key: start, Timestamp: resumeKeyTS},
-				EndKey:             args.EndKey,
-				StartTS:            args.StartTime,
-				EndTS:              h.Timestamp,
-				ExportAllRevisions: exportAllRevisions,
-				TargetSize:         targetSize,
-				MaxSize:            maxSize,
-				MaxIntents:         maxIntents,
-				StopMidKey:         args.SplitMidKey,
-				ResourceLimiter:    storage.NewResourceLimiter(storage.ResourceLimiterOptions{MaxRunTime: maxRunTime}, timeutil.DefaultTimeSource{}),
-			}, destFile)
+		opts := storage.MVCCExportOptions{
+			StartKey:           storage.MVCCKey{Key: start, Timestamp: resumeKeyTS},
+			EndKey:             args.EndKey,
+			StartTS:            args.StartTime,
+			EndTS:              h.Timestamp,
+			ExportAllRevisions: exportAllRevisions,
+			TargetSize:         targetSize,
+			MaxSize:            maxSize,
+			MaxIntents:         maxIntents,
+			StopMidKey:         args.SplitMidKey,
+			ResourceLimiter:    storage.NewResourceLimiter(storage.ResourceLimiterOptions{MaxRunTime: maxRunTime}, timeutil.DefaultTimeSource{}),
+		}
+		var summary roachpb.BulkOpSummary
+		var resume storage.MVCCKey
+		var fingerprint uint64
+		var err error
+		if args.ExportFingerprint {
+			summary, resume, fingerprint, err = storage.MVCCExportFingerprint(ctx, cArgs.EvalCtx.ClusterSettings(), reader, opts, destFile)
+		} else {
+			summary, resume, err = storage.MVCCExportToSST(ctx, cArgs.EvalCtx.ClusterSettings(), reader, opts, destFile)
```

Comment:
💭 I wonder if the tenant-eliding options should be the default or of we should thread those through the command args and allow the user to specify them.

Reply:
I think this is a good point, I can't think of a use case where someone wants to keep the tenant prefix and value checksum. I'm leaning towards defaulting to strip everything, and if we need it going forward we can always thread it through?

Reply:
I also just realized I'll need to strip the tenant prefix and value checksum when fingerprinting range keys.

Reply:
Okay I've set the default to true, and refactored things to strip the key/value checksum in rangekey fingerprinting. Let me know what you think!